ivy/functional/frontends/paddle/nn/functional/loss.py

```diff
@@ -274,6 +272,29 @@ def nll_loss(
     """Refer
     https://pytorch.org/docs/stable/generated/torch.nn.NLLLoss.html#torch.nn.NLLLoss for
     more on NLL(Negative log likelihood) Loss."""
+    if reduction not in ["sum", "mean", "none"]:
+        raise ValueError(
```

Comment:
For all the value errors I would sugges to  `ivy.utils.exceptions.IvyValueError` see [0, 1].

[0] https://unify.ai/docs/ivy/docs/utils/ivy.utils.exceptions.html#ivy.utils.exceptions.IvyValueError
[1] https://unify.ai/docs/ivy/overview/deep_dive/exception_handling.html